Raids/Prophecy of Ahnket/LFR/Tartaric_Depths/Council.lua

```diff
@@ -191,9 +194,10 @@ COF.Lang.Ability.Void = KBM.Language:Add("Void")
 
 -- Verbose Dictionary
 COF.Lang.Verbose = {}
-COF.Lang.Verbose.Flamescape = KBM.Language:Add("Go into the circle")
-COF.Lang.Verbose.Flamescape:SetGerman("In den weissen Kreis")
-COF.Lang.Verbose.Flamescape:SetFrench("Aller dans le cercle")
+COF.Lang.Verbose.Flamescape = KBM.Language:Add("Flamescape")
```

Comment:
We no longer need the `Verbose` version, we can directly use the ability name from `COF.Lang.Ability`